src/datanode/src/tests/instance_test.rs

```diff
@@ -639,6 +639,55 @@ async fn test_use_database() {
     check_output_stream(output, expected).await;
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn test_delete() {
```

Comment:
Can you also add some cases to our [sqlness](https://github.com/GreptimeTeam/greptimedb/tree/develop/tests) suite? Maybe in a new folder `standalone/delete`.

Reply:
Done, In addition, it also fixes the problem that cannot be repeated(`insert_invalid.sql`) 
`schema.sql` is also cannot be repeated, but `DROP SCHEMA test_public_schema;` is not supported.